.gitlab-ci.yml

```diff
@@ -0,0 +1,22 @@
+stages:
+  - trigger
+
+
+trigger-veracode-scan:
+  # prepare and submit for static code analysis
+  stage: trigger
+  only:
+    variables:
+      - $VERACODE_API_ID
+  image: openjdk:8
+  before_script:
+    # The latest wrapper version can be found in https://repo1.maven.org/maven2/com/veracode/vosp/api/wrappers/vosp-api-wrappers-java/
+    - wget -q -O veracode-wrapper.jar https://repo1.maven.org/maven2/com/veracode/vosp/api/wrappers/vosp-api-wrappers-java/${VERACODE_WRAPPER_VERSION}/vosp-api-wrappers-java-${VERACODE_WRAPPER_VERSION}.jar
+    - tar -f scan.tar --append ota-plus-web/app/reactapp
+  script:
+    - java -jar veracode-wrapper.jar -vid ${VERACODE_API_ID} -vkey ${VERACODE_API_KEY}
+      -action UploadAndScan -appname "OTA Frontend" -createprofile true -autoscan true
```

Comment:
Is this a multiline string, or strange indentation? either way I think yaml might complain...

Reply:
It's multiline and gitlab is fine with it - see linked test run. Also, it's copy-paste from existing job in another project.

Reply:
Huh, guess I learned yet another weird thing about yaml arrays, thanks!